### DISCUSSION TITLE
Doesn't match any Serverless Functions inside the api directory

### DISCUSSION THREAD

**Discussion by senthilsivanath** [vercel/community#3990] [Help]
### Question

Trying to increase the memory limit of a server less function with vercel.json but glob pattern is not able to match any server less function in api directory. Have tried with several combinations around the path, but I am getting the following error

```
Error: The pattern "src/api/test.js" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.
```

Function is available in 
```src/api/test.js```

Vercel config used
```json5
{ 
  "functions": {     
  "api/test.js": {
    "memory": 3008,
    "maxDuration": 10        
  }
}
}
```

**Comment by amyegan** [comment]
Hey @senthilsivanath. We need to know more about your app to be able to make configuration suggestions. Can you share the file structure or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)?

**Comment by senthilsivanath** [comment]
Its a gatsby project with a server less function.

**Comment by smartraysam** [comment]
I am using remix framework 
i have my serverless function in app/routes/api.chat.js "

am getting error
The pattern "app/routes/api.chat.tsx" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.

**Reply by amyegan** [reply]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to get a better view of what configuration is needed?

**Comment by ezzabuzaid** [comment]
```json title=vercel.json
{
  "$schema": "https://openapi.vercel.sh/vercel.json",
  "version": 2,
  "installCommand": "npm install --only=production",
  "buildCommand": "npm run build:remote",
  "devCommand": "npm run start:prod",
  "outputDirectory": "build",
  "functions": {
    "build/server.js": {
      "runtime": "@vercel/node@3.1.5",
      "maxDuration": 15
    }
  }
}
```

**_Error: The pattern "build/server.js" defined in `functions` doesn't match any Serverless Functions inside the `api` directory._**


any help?
